app/views/idv/shared/_error.html.erb

```diff
@@ -9,16 +9,14 @@ locals:
 * options: Array of troubleshooting options.
 %>
 <% if local_assigns.fetch(:type, :error) == :error
-     image_src = 'status/error.svg'
+     icon_name = 'error'
      troubleshooting_heading = t('idv.troubleshooting.headings.need_assistance')
-     alt = t('errors.alt.error')
-   else
-     image_src = 'status/warning.svg'
-     troubleshooting_heading = t('components.troubleshooting_options.default_heading')
-     alt = t('errors.alt.warning')
-   end %>
+else
+  icon_name = 'warning'
```

Comment:
Yes, we should make this a symbol explicitly. However, this does not fail currently because we ensure that the first argument to the initializer is always a symbol [here](https://github.com/18F/identity-idp/pull/6983/files/b55057b68d1cbf922b304d1e2330f018437c9b4c#diff-d57d77d0ef6739fea5457ddbf5acb0f37ce15d6097801b23a91ec9db2f999fc1R14)

Reply:
Ah, I had missed that. Still, I think given the [other problematic case](https://github.com/18F/identity-idp/pull/6983#discussion_r980068883), having validation with a raised exception could allow us to remove that graceful casting, if we wanted symbols to be the preferred argument type.